schedule_scaling/main.py

```diff
@@ -66,27 +66,36 @@ def parse_schedules(schedules, identifier):
         return []
 
 
-def get_delta_sec(schedule):
+def get_delta_sec(schedule, now):
     """ Returns the number of seconds passed since last occurence of the given cron expression """
-    # get current time
-    now = datetime.now()
-    # get the last previous occurrence of the cron expression
-    time = croniter(schedule, now).get_prev()
+    # get the last previous occurrence of the cron expression, add 1 second here to handle cases
+    # when schedule should be executed now and delta is 0, now is rounded to full minutes, so it's
+    # ok anyway
+    time = croniter(schedule, now + timedelta(seconds=1)).get_prev()
```

Comment:
Now times are rounded up to a minute and we can't rely on any nanoseconds added by `datetime.now()` to make `get_prev()` return current minute...

More specifically: before not-rounded datetime.now() was used here and it was equal to 'BASE_MINUTE + same seconds / some milliseconds', those seconds / milliseconds made `get_prev` return current minute when schedule was happening now (like, current minute is 16:30, schedule is "30 16 * * *", now returns "16:30:00.100). With my changes now is rounded up to a minute so I have to add this margin explicitly